frontend/src/tabs/zoom.pug

```diff
@@ -1,17 +1,19 @@
 #zoom
   span.large-font Commits Panel
   .toolbar(v-if="filteredUser.commits.length")
-    a(v-if="expandedCommitMessagesCount === 0" v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages
-    a(v-else v-on:click="toggleAllCommitMessagesBody(false)") hide all commit messages
+    a(v-if="expandedCommitMessagesCount === 0", v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages
```

Comment:
Didn't this line violates the maximum character rule?

Reply:
It is going up to 118, so didn't.